app/models/job_proxy_dispatcher.rb

```diff
@@ -90,6 +91,36 @@ def dispatch
     _log.info "Complete - Timings: #{t.inspect}"
   end
 
+  def dispatch_container_jobs
+    jobs_to_dispatch, = Benchmark.realtime_block(:pending_container_jobs) { pending_jobs(ContainerImage) }
+    Benchmark.current_realtime[:container_jobs_to_dispatch_count] = jobs_to_dispatch.length
+
+    concurrent_container_scans_limit = Settings[:container_scanning][:concurrent_per_ems].to_i
+    if jobs_to_dispatch.length > 0
+      jobs_to_dispatch.each do |job|
+        if concurrent_container_scans_limit > 0 &&
+           active_scans(ContainerImage)[@zone] >= concurrent_container_scans_limit
+          _log.warn "SKIPPING remaining container image scan jobs in dispatch since there are "\
+                    " [#{@active_scans[@zone]}]"" active scans in the zone [#{@zone}]"
```

Comment:
this `@active_scans` is wrong because it's not accounting for the class of scans.  We only want to warn about the container image scans currently being run.